test/unit/views/map.cpp

```diff
@@ -24,7 +24,7 @@ struct store_op
 {
   auto operator()(auto x, eve::as<int>) const
   {
-    return eve::convert(x / 2.0, eve::as<int>{});
+    return eve::convert(x / 2.0, eve::as<double>{});
```

Comment:
No, this is wrong

Reply:
done

Reply:
I mean - it's still in it's incorrect shape.